.travis.yml

```diff
@@ -10,9 +12,12 @@ jobs:
     - stage: test
       name: "\U0001F5DC️ Run tests"
       script: npm run test:coverage
+    - stage: test
+      name: "📦 bundlesize"
+      script: npm run profile
     - stage: publish
-      name: Publish to npm
+      name: `🚀 Publish to npm'
```

Comment:
@dangodev could this be the reason Travis won't run? Looks like we have mismatched quotes so maybe the yaml can't be parsed.

Reply:
🤦‍♂